From 268a3efd9843f3146c03941b1a7165d1c3a72148 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Tue, 15 Aug 2017 15:34:55 -0500
Subject: [PATCH 247/278] drivers/pci: Reduce the Axxia LOS Timeout

When using the LOS work around on Axxia parts, fail
if there is no link in 1 second (formerly 2 seconds).

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 drivers/pci/host/pcie-axxia.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/pci/host/pcie-axxia.c b/drivers/pci/host/pcie-axxia.c
index fd8eb54..5f8248a 100644
--- a/drivers/pci/host/pcie-axxia.c
+++ b/drivers/pci/host/pcie-axxia.c
@@ -1134,9 +1134,13 @@ int axxia_pcie_link_up(struct pcie_port *pp)
 			break;
 		}
 
+		/*
+		 * Give up if there is no link after 1 second.
+		 */
+
 		do_gettimeofday(&now);
 
-		if ((2 * 1000 * 1000) <
+		if ((1000 * 1000) <
 		    (((now.tv_sec * 1000 * 1000) + now.tv_usec) -
 		     ((start.tv_sec * 1000 * 1000) + start.tv_usec))) {
 			rc = -1;
-- 
1.8.5.2.233.g932f7e4

